aeron-client/src/main/cpp/concurrent/logbuffer/LogBufferDescriptor.h

```diff
@@ -269,9 +270,17 @@ inline void endOfStreamPosition(AtomicBuffer &logMetaDataBuffer, std::int64_t po
     logMetaDataBuffer.putInt64Ordered(LOG_END_OF_STREAM_POSITION_OFFSET, position);
 }
 
+inline std::int32_t computeTermCount(
+    std::int32_t termId,
+    std::int32_t initialTermId) noexcept
+{
+    const std::int64_t difference = static_cast<std::int64_t>(termId) - static_cast<std::int64_t>(initialTermId);
+    return static_cast<std::int32_t>(difference & 0xFFFFFFFF);
+}
```

Comment:
This change needs to be copied across to the wrapper.

Reply:
Done but frankly speaking I don't understand why we have `LogBufferDescriptor.h` in the wrapper in the first place.